Sources/Commandant/Errors.swift

```diff
@@ -10,9 +10,9 @@ import Foundation
 import Result
 
 #if swift(>=3)
-	public typealias ClientErrorType = ErrorProtocol
+	public typealias ClientErrorType = Error
 #else
-	public typealias ClientErrorType = ErrorType
+	public typealias ClientErrorType = ErrorProtocol
```

Comment:
I don't think `Swift 2.x` is supported after all these changes, so I'd just remove this.